src/views/Bucket.vue

```diff
@@ -63,10 +64,19 @@ export default {
         start: this.daterange[0].format(),
         end: this.daterange[1].format(),
       });
+      this.events = this.bucket_with_events.events;
     },
     getEventCount: async function(bucket_id) {
       this.eventcount = (await this.$aw.countEvents(bucket_id)).data;
     },
+    updateEvent: function(event) {
+      const i = this.events.findIndex((e) => e.id == event.id);
+      if(i != -1) {
+        this.$set(this.events, i, event);
```

Comment:
Would be nice with a comment like I did in my previous change why this.$set is needed and just modifying this.events is not sufficient. It's an detail that is easy to miss otherwise.